src/server/pom.xml

```diff
@@ -274,7 +274,7 @@
         <dependency>
             <groupId>io.k8ssandra</groupId>
             <artifactId>datastax-mgmtapi-client-openapi</artifactId>
-            <version>0.1.0-507f418</version>
+            <version>0.1.0-c22a2fc</version>
```

Comment:
this is the version published from the latest commit on Management API Integration branch:
https://github.com/k8ssandra/management-api-for-apache-cassandra/tree/feature/client-gen
commit: https://github.com/k8ssandra/management-api-for-apache-cassandra/commit/c22a2fc7127097a7a20c6d0ad4bb5d117a6ea75a